src/universal/utils/hasChildRoutes.js

```diff
@@ -0,0 +1,3 @@
+const hasChildRoutes = (module) => (module ? !!module.childRoutes : false);
```

Comment:
```suggestion
const hasChildRoutes = (module) => (module && !!module.childRoutes);
```

Reply:
This change would not work if we want return values to be a boolean. But, solution will still work.

Reply:
why does it matter ? if there is a module it returns a boolean. if not its falsy